### PR TITLE
allow NAs again (for real now)

### PR DESCRIPTION
**Pull request by dpprdan**
This PR adds support for `NA`s in placenames and latitude longitude again. Queries with `NA`s are not sent to the API. An empty/`NA` response will be returned depending on the specific query.

### CODE REVIEW

**Comment by dpprdan** [comment]
The codecov warnings are false positives due to the missing key on PRs (probably should disable those tests for PRs for now). I get 98.30% coverage locally.

File: R/oc_bbox.R [hunk 1]
```diff
@@ -131,16 +131,16 @@ oc_check_bbox <- function(xmin, ymin, xmax, ymax) {
   if (!all(is.numeric(bnds))) {
     stop("Every `bbox` must be a numeric vector.", call. = FALSE)
   }
-  if (!dplyr::between(xmin, -180, 180)) {
```
Comment (maelle) [inline R/oc_bbox.R:134]: why not an helper function for that? `opencage_between()` or so?

**Review by maelle — APPROVED** [review]
LGTM! Is it an use case you encounter? (I guess so :sweat_smile: )

File: R/oc_bbox.R (hunk 1 above)
Comment (dpprdan) [reply R/oc_bbox.R:134]: I made this its own check function... 🤷🏻‍♂️

**Comment by dpprdan** [comment]
> Is it an use case you encounter?

Sometimes I might have a not-so-clean dataset that I want to geocode quickly. Then it helps if it doesn't abort the whole thing in case there are only a few NAs...

File: tests/testthat/test-oc_check_query.R [hunk 2]
```diff
@@ -257,3 +257,17 @@ test_that("oc_check_logical checks no_record", {
     )
   )
 })
+
+# Test oc_check_between() -------------------------------------------------
+test_that("oc_check_logical checks no_record", {
+  # numeric()
+  symbol <- 11
+  expect_error(oc_check_between(symbol, 0, 10))
```
Comment (maelle) [inline tests/testthat/test-oc_check_query.R]: you could use `expect_snapshot_error()`.

**Review by maelle — APPROVED** [review]
:rocket:

File: tests/testthat/test-oc_check_query.R (hunk 2 above)
Comment (dpprdan) [reply tests/testthat/test-oc_check_query.R]: using `expect_snapshot(error = TRUE)` now, so the snap file also contains the code and not only the error message (which it does when using `expect_snapshot_error()`

**Pull request merged**